Rebuild-Black-Business/RBB-Website
# [feature] Add Business filtering

# Describe your PR

This PR adds a new filtering component to `/businesses` that allows users to filter businesses by:
1. Type
2. Location (zip or city/state) string
3. Whether the business was effect by protests or not

## Pages/Interfaces that will change
The `BusinessFilter` component will only appear on the `/businesses` route for now.

### Screenshots / video of changes

https://www.loom.com/share/fc78caaba9a7474b86bc41ae71d6ce02

## Steps to test

1. Go to the `/businesses` route
2. Try out the filters
3. Confirm the data is being re-filtered
